spyder/plugins/application/container.py

```diff
@@ -326,6 +324,8 @@ def _check_updates_ready(self):
                     self.application_update_status.start_installation(
                         latest_release=latest_release)
 
+                return
```

Comment:
I don't think we can have this `return` statement since code on lines 419-425 must be executed after this `elif update_available:` block.

Reply:
You're right. I moved this `return` to be inside the `if` block above, which is where it really belongs.